### PR TITLE
Dev 0.8.8

### PR DESCRIPTION
**Pull request by mmzdouc**
Merge additional features and quality-of-life imrovements into main.

### CODE REVIEW

**Review by mmzdouc — COMMENTED** [review]
Code reviewed. 
Tested on Ubuntu Linux and Windows.

File: src/fermo/pages/pages_dashboard.py [hunk 1]
```diff
@@ -969,6 +1010,97 @@ def call_feature_export_button():
             'width' : '60%'},
         )
 
+
+
```
Comment (shabicht) [inline src/fermo/pages/pages_dashboard.py:1014]: just to test the comment function: why are there so many empty lines here?

**Comment by shabicht** [comment]
Assuming the review looks the same for you as for me, the "checks" tab contains the tests that are run automatically, when you push a commit, right?

File: src/fermo/pages/pages_dashboard.py (hunk 1 above)
Comment (mmzdouc) [reply src/fermo/pages/pages_dashboard.py:1014]: No reason, could be removed. Linting and automatic formatting is not implemented yet but will be in a future version.

**Pull request merged**